### PR TITLE
fix: add needed env variables for local testing

### PR DESCRIPTION
**Pull request by vasco-santos**
My local env was not working and after digging through the dev deployment to find out what was going on, noticed that there were no docs for the Delegations Bucket, so added it now.

Also ended up adding a validation for the environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,10 @@ Bucket name to replicate written .idx files.
 
 Bucket name to replicate root CID to car CIDs mapping.
 
+#### `R2_DELEGATION_BUCKET_NAME`
```
Comment (olizilla) [inline README.md:143]: Do we need to describe `PROVIDERS` here in the read me as well?

**Review by olizilla — APPROVED** [review]

File: .env.tpl [hunk 2]
```diff
@@ -14,6 +14,7 @@
 EIPFS_INDEXER_SQS_ARN = 'arn:aws:sqs:us-west-2:505595374361:staging-ep-indexer-topic'
 EIPFS_INDEXER_SQS_URL = 'https://sqs.us-west-2.amazonaws.com/505595374361/staging-ep-indexer-topic'
 
+PROVIDERS = ''
 ACCESS_SERVICE_DID = ''
```
Comment (olizilla) [inline .env.tpl:18]: is `ACCESS_SERVICE_DID` used anywhere?

Comment (vasco-santos) [reply .env.tpl]: looks like no, good catch. Removing it

**Comment by travis** [comment]
fabulous, thank you!!

**Review by travis — APPROVED** [review]

**Pull request merged**